ios/sdk/WeexSDK/Sources/Module/WXAnimationModule.m

```diff
@@ -355,7 +360,14 @@ - (void)_createCAAnimation:(WXAnimationInfo *)info
         layer.anchorPoint = CGPointZero;
         layer.frame = originFrame;
     }
-    [layer addAnimation:animation forKey:info.propertyName];
+    
+    if(WXFloatEqual(animation.duration, 0) || WXFloatLessThan(animation.duration, 0)){
```

Comment:
may be we can provide more function like lessOrEqualThan  and  and greatOrEqualThan value, here will be more less code